openmc/data/endf.c

```diff
@@ -0,0 +1,37 @@
+#include <stdlib.h>
+
+double cfloat_endf(const char* buffer, int n)
+{
+  char arr[12]; // 11 characters plus a null terminator
+  int j = 0; // current position in arr
+  int found_decimal = 0;
+  int found_exponent = 0;
+  for (int i = 0; i < n; ++i) {
+    // Skip whitespace characters
+    char c = buffer[i];
+    if (c == ' ') continue;
+
+    if (found_decimal) {
```

Comment:
What about exponentials without decimals?  I think this would miss e.g. `1+3`

Reply:
Ah, you're right. Thanks for catching that!